icu4c/source/test/depstest/dependencies.txt

```diff
@@ -979,6 +980,8 @@ group: number_output
     number_representation format formatted_value_sbimpl
     # PluralRules internals:
     unifiedcache
+    # Unit Formatting
```

Comment:
This comment isn't really correct.  The dependency is because FormattedNumber now holds a MeasureUnit by value.  It doesn't necessarily have to do with formatting.

Reply:
Done.